.github/workflows/codeql.yml

```diff
@@ -0,0 +1,51 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  workflow_dispatch:
+  schedule:
+    - cron: '15 14 * * 1'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-20.04
+    container: seladb/ubuntu2204
+    permissions:
+      actions: read
+      contents: read
```

Comment:
According to the documentation, this is only needed for private repos...

Reply:
Sorry, I don't follow. `Permissions:` at all or some specific `right`?

Reply:
According to the CodeQL Github Actions README file:

![image](https://user-images.githubusercontent.com/9059541/187063380-f10244d1-68fc-4742-b53a-a213006f2448.png)

https://github.com/github/codeql-action#usage

Reply:
I see. I'll remove it.